src/components/editor/EditorArticleController.jsx

```diff
@@ -56,7 +57,7 @@ export default class EditorArticleController extends FalcorController {
   }
 
   static getFalcorPathSets(params) {
-    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']]];
+    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']], ['latestIssue']];
```

Comment:
What if instead, we made issues an array, that is ordered. I think this makes sense anyway, cause issues have an order. Just give an issue a creationDate property, and always order by that?